separate_instance/database/Dockerfile

```diff
@@ -0,0 +1,128 @@
+##################################################################################
```

Comment:
whats the change in this file?

Reply:
...
LABEL org.label-schema.name="gradido:database2"
LABEL org.label-schema.description="Gradido Database Migration Service 2"
...
LABEL org.label-schema.vendor="Gradido Community 2"
...